src/perlmodgen.cpp

```diff
@@ -1520,8 +1520,8 @@ void PerlModGenerator::generatePerlModForMember(const MemberDef *md,const Defini
     .addFieldQuotedString("protection", getProtectionName(md->protection()))
     .addFieldBoolean("static", md->isStatic());
 
-  addPerlModDocBlock(m_output,"brief",md->getDefFileName(),md->getDefLine(),md->getOuterScope(),md,md->briefDescription());
-  addPerlModDocBlock(m_output,"detailed",md->getDefFileName(),md->getDefLine(),md->getOuterScope(),md,md->documentation());
+  addPerlModDocBlock(m_output,"brief",md->getDefFileName(),md->briefLine(),md->getOuterScope(),md,md->briefDescription());
```

Comment:
If we change this to use `md->briefLine()` then also `md->getDefFileName()` should be changed to  `md->briefFile()` (brief description can be in a header and `getDefFileName()` can point to the source file).

Reply:
Indeed